recipes/nettle/all/test_package/CMakeLists.txt

```diff
@@ -1,8 +1,7 @@
 cmake_minimum_required(VERSION 3.1)
```

Comment:
Should the minimum CMake version here be higher?

Reply:
Not in this case. We are not using `target_compile_features` which could require 3.8.

Reply:
@jwillikers CMakeToolchain will ask 3.15 indeed, but for that CMakeLists we only need 3.1.

CMake 3.15 has been determined as default version: https://github.com/conan-io/tribe/blob/main/design/004-tools-cmake.md

Reply:
Thanks for the link! I see that the `conan_toolchain.cmake` checks that at least version `3.15` is in use, too. I guess I've been specifying `3.15` in test packages since that lines up with the minimum version required by `CMakeToolchain`. So the `CMakeToolchain` doesn't require any CMake Policy changes from 3.15?